tools/make_patch.c

```diff
@@ -168,7 +180,7 @@ struct symbol *parse_symfile(FILE *file, const char *prefix)
 	buffer = create_buffer();
 	if (!buffer) goto error;
 
-	// Parse the symfile, to get the symbols we want.
+	// Parse the entire symfile, to the the symbols we want.
```

Comment:
```suggestion
	// Parse the entire symfile, to gather the symbols
```

Reply:
Resolved in https://github.com/pret/pokecrystal/pull/882/commits/3a78bcc8073928f2675d66b38124b90b6796ffca